src/components/Editor/Guide.tsx

```diff
@@ -0,0 +1,100 @@
+import { useContext } from "react";
+import Backdrop from '@mui/material/Backdrop';
+import Box from '@mui/material/Box';
+import Modal from '@mui/material/Modal';
+import Fade from '@mui/material/Fade';
+import Grid from "@mui/material/Grid";
+import ArrowLeftIcon from "@mui/icons-material/ArrowLeft";
+import ArrowRightIcon from "@mui/icons-material/ArrowRight";
+import Typography from '@mui/material/Typography';
+import { useGuide } from "../../util/customHooks/guideFunc"
+import { SideBarContext } from "../../util/customHooks/SideBarContext";
+
+const style = {
+  position: 'absolute',
+  top: '50%',
+  left: '80%',
+  transform: 'translate(-50%, -50%)',
+  width: 400,
+  bgcolor: 'background.paper',
+  background: '#3f3d56',
+  borderRadius: '15px',
+  boxShadow: 24,
+  p: 4, 
+  fontFamily: 'Trebuchet MS'
+};
+
+function Guide() : JSX.Element {
+    const { 
+        guideNum,
+        showGuide, 
+        toggleGuide, 
+    } = useContext(SideBarContext);
+
+    const { pages, prevPage, nextPage, renderIcon } = useGuide();
+
+    return (
+        <div>
+            <Modal
+            aria-labelledby="transition-modal-title"
+            aria-describedby="transition-modal-description"
+            open={showGuide}
+            onClose={toggleGuide}
+            closeAfterTransition
+            BackdropComponent={Backdrop}
+            BackdropProps={{
+                timeout: 500,
+            }}
+            >
+            <Fade in={showGuide}>
+                <Box sx={style}>           
```

Comment:
the sx is not passing, 
It's a problem with typescript. By having style directly typed in side will be fine.
i.g. sx ={{position: 'absolute',
  top: '50%',
  left: '80%',
  transform: 'translate(-50%, -50%)',
  width: 400,
  bgcolor: 'background.paper',
  background: '#3f3d56',
  borderRadius: '15px',
  boxShadow: 24,
  p: 4, 
  fontFamily: 'Trebuchet MS'
  }}

Reply:
Kinda odd that typescript can't handle this but it should be fixed now.